connectivity/perf/common/metrics.go

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package common
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"time"
+)
+
+type LatencyMetric struct {
```

Comment:
nit: no docs on exported defs and methods

Reply:
Fixed.